packages/dds/tree/src/schema-view/README.md

```diff
@@ -0,0 +1,9 @@
+# Schema-View
+
+This folder extends "Schema-Stored", adding APIs for applications to view and interact with documents which use stored-schema.
```

Comment:
Why is it called schema-stored in some places and stored-schema in others?

Reply:
The package is named "schema-stored" so that someone looking for schema stuff can easily find it (and schema-view): for identifiers like package names, it's nice for them to go from less specific to more specific from left to right (this makes related things sort together, makes autocomplete work better, gives the name the same structure they would have if we introduced more hierarchy (ex: a schema folder). Effectively its big-endian word order.

In English prose however, adjectives come first, and in this case it's the noun that's the most significant for these names.

I intentionally took the approach of using both conventions in different places. I don't like the inconsistency, but it seems like a reasonable option. Personally (and subjectively) I feel what it did is slightly better than use just one or the other everywhere: this is not a strong opinion.

If we wanted to use the natural order for our language, and have good grouping, I guess we could make the more significant word into an adjective (ex: schematizing-persistent-data and schematizing-view). I hadn't really thought of this approach before: I don't think it works well here though, but might be useful in other places. Or we could just do development in in a language that puts nouns first like Spanish.

Reply:
That makes sense. I did have a bit of confusion cause view could also be a noun, vs stored which is obviously an adjective, so I wasn't sure if view schema and schema view were actually two separate things. Is there an equivalent adjective for view that we could use?